java/src/org/openqa/selenium/grid/node/k8s/OneShotNode.java

```diff
@@ -113,14 +115,16 @@ private OneShotNode(
       URI uri,
       URI gridUri,
       Capabilities stereotype,
-      WebDriverInfo driverInfo) {
+      WebDriverInfo driverInfo,
+      SlotMatcher slotMatcher) {
```

Comment:
We should pass a `Predicate` here, in the same way it was done at the `LocalNodeFactory`.

Reply:
I completely removed slotMatcher as it's not used in class